hoomd_benchmarks/write_gsd_log.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021-2023 The Regents of the University of Michigan
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Benchmark GSD writes."""
+
+import hoomd
+from . import write_gsd
+
+
+class GSDLog(write_gsd.GSD):
+    """Log-only GSD benchmark.
+
+    Args:
+        kwargs: Keyword arguments accepted by ``GSD.__init__``
+
+    See Also:
+        `write_gsd.GSD`
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self.bytes_per_step = 28 / 1024**2
```

Comment:
We appear to be logging only a single scalar, which should only be 4 bytes if I'm not mistaken.

Reply:
I'm accounting for the 24 bytes of index as well.

Reply:
Perhaps a short comment could clarify this to avoid future confusion.